### PR TITLE
kubeadm cleanup: master -> control-plane (cont.4)

### PR DESCRIPTION
**Pull request by vanduc95**
This PR continuation of [#73987](https://github.com/kubernetes/kubernetes/pull/73987)

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```
/assign @neolit123

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
thanks @vanduc95 !!

/kind cleanup
/ok-to-test
/priority backlog
/approve

File: cmd/kubeadm/app/phases/addons/dns/manifests.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ spec:
             path: /readiness
             port: 8081
             scheme: HTTP
-          # we poll on pod startup for the Kubernetes master service and
+          # we poll on pod startup for the Kubernetes control-plane service and
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/addons/dns/manifests.go]: @vanduc95 
on lines 176 and 244 in this file, MasterTaintKey has to be modified too.

this e2e test caught that:
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/74473/pull-kubernetes-e2e-kind/5/build-log.txt

> phase addon/coredns: error when parsing CoreDNS deployment template: error when executing template: template: template:28:16: executing \"template\" at <.MasterTaintKey>: can't evaluate field MasterTaintKey in type struct { DeploymentName string; Image string; ControlPlaneTaintKey string }"

Comment (neolit123) [reply cmd/kubeadm/app/phases/addons/dns/manifests.go]: /hold

Comment (vanduc95) [reply cmd/kubeadm/app/phases/addons/dns/manifests.go]: Thanks @neolit123, I have updated.

**Pull request merged**